Sources/SnapshotAssertion/SnapshotAssertion.swift

```diff
@@ -4,8 +4,8 @@ var diffTool: String? = nil
 
 public protocol Diffable: Equatable {
   static var diffableFileExtension: String? { get }
+  static func fromDiffableData(_ data: Data) -> Self
```

Comment:
Non-final classes (like `UIImage`) have trouble conforming to protocol-based `convenience init`s.